UPGRADE.md

```diff
@@ -84,7 +126,7 @@ This process can take a while to complete, depending on the number/size of the L
 
 If you would like to verify that your new learning materials are correctly stored, you can run the following command when the migration is complete:
 
-```bash
+ ```bash
```

Comment:
There should be no space before the first tick-mark (`), otherwise markdown doesn't render the syntax...